packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue

```diff
@@ -13,24 +12,18 @@ import type { ExecutionSummary } from 'n8n-workflow';
 import { useWorkflowsStore } from '@/stores/workflows.store';
 import { useExecutionsStore } from '@/stores/executions.store';
 
-const props = defineProps({
-	executions: {
-		type: Array as PropType<ExecutionSummary[]>,
-		default: () => [],
+const props = withDefaults(
+	defineProps<{
+		executions: ExecutionSummary[];
+		filters: ExecutionFilterType;
+		total: number;
+		estimated: boolean;
+	}>(),
+	{
+		total: 0,
+		estimated: false,
 	},
-	filters: {
-		type: Object as PropType<ExecutionFilterType>,
-		default: () => ({}),
```

Comment:
Missed the default here.

Reply:
these properties are required now, so no default needed. I checked the usage of many (but not all) of these components and updated some props to be required if we were passing them in from everywhere else in the code.

Reply:
Good, thanks for checking. Just wanted to make sure 👍